help/md/power.md

```diff
@@ -1,9 +1,18 @@
 ## name
 power
 ## title
+power
 ## description
+The function raises any real number to the power of x.
```

Comment:
Maybe just "to a power"?   It mostly clear what you mean, but technically its not clear what `x` is.